src/sentry/rules/actions/base.py

```diff
@@ -109,15 +131,9 @@ def _linked_issues(event, integration):
     )
 
 
-def get_linked_issue_ids(event, integration):
-    return _linked_issues(event, integration).values_list("key", flat=True)
-
-
-def has_linked_issue(event, integration):
-    return _linked_issues(event, integration).exists()
-
-
-def create_link(integration, installation, event, response):
+def create_link(
+    integration: Integration, installation: Any, event: Event, response: Response
```

Comment:
The docs say that an `installation` is an `Installation` object. Is that not true? Why is `installation` typed as `Any`?